frontend_app_file/src/container/File.jsx

```diff
@@ -167,7 +169,8 @@ export class File extends React.Component {
       newContent: newContentObject,
       editionAuthor: data.author.public_name,
       hasUpdated: prev.loggedUser.userId !== data.author.user_id,
-      timeline: addRevisionFromTLM(data, prev.timeline, prev.loggedUser.lang)
+      timeline: addRevisionFromTLM(data, prev.timeline, prev.loggedUser.lang),
+      isLastTimelineItemCurrentToken: data.client_token === getOrCreateSessionClientToken()
```

Comment:
instead of calling the function every time, you can have a variable that store it in the component.
It wont change. I think a class property can be a good place